src/frame/worker/GroupCrypto.ts

```diff
@@ -11,17 +11,17 @@ import {publicKeyToBytes} from "../../lib/Utils";
  * @param {SigningKeyPair}        signingKeys   Current users signing keys used to sign transform key
  * @param {boolean}               addAsMember   Whether to add the current group admin as a member
  */
-export function createGroup(userPublicKey: PublicKey<Uint8Array>, signingKeys: SigningKeyPair, addAsMember: boolean) {
+export function createGroup(userPublicKey: PublicKey<Uint8Array>, signingKeys: SigningKeyPair, memberList?: UserOrGroupPublicKey[]) {
```

Comment:
Similar to the `transformKeyGrantList` change to make that a fixed array, it's probably beneficial to do the same thing here and make this `memberList` not be optional. That'll clean up the code below to not have to check whether it's undefined or not. That change should also spider out to the message passing that happens between the frame and the worker but I don't think it'll change the code that builds up that message because it's already always being passed as an array, it just might be empty.

Reply:
done